### PR TITLE
Kbd6x

### PR DESCRIPTION
**Pull request by peott-fr**
## Description

Adding a simple KBD6x keymap. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/primekb/prime_e/keymaps/peott-fr/keymap.c [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  // set Layer 1 to output and low
+  setPinOutput(B2);
+  writePinLow(B2);
+  // set Layer 2 to output and low
+  setPinOutput(B3);
+  writePinLow(B3);
+}
+
+void matrix_scan_user(void) {
+}
```
Comment (drashna) [inline keyboards/primekb/prime_e/keymaps/peott-fr/keymap.c:71]:
```suggestion
```

Comment (peott-fr) [reply keyboards/primekb/prime_e/keymaps/peott-fr/keymap.c]: This file shouldn't have been in that branch. I deleted it as its own branch's PR got approved.

**Review by fauxpark — APPROVED** [review]
Could do with some formatting, but LGTM

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**